packages/linter/src/migrations/update-11-5-0/always-use-project-level-tsconfigs-with-eslint.ts

```diff
@@ -40,7 +40,11 @@ function updateProjectESLintConfigs(host: Tree) {
       return;
     }
 
-    const parserOptionsProjectVal = [`${p.root}/tsconfig.*?.json`];
+    const isNextJs = p.targets?.build?.executor === '@nrwl/next:build';
+
+    const parserOptionsProjectVal = isNextJs
+      ? [`${p.root}/tsconfig(.*)?.json`]
```

Comment:
this is related to a change that was introduced in #4970 , where Next.js apps need this path to be different from other apps